src/main/resources/api/openapi.yaml

```diff
@@ -303,6 +307,282 @@ components:
           description: Sigma source to be compiled
           type: string
 
+    ErgoLikeTransaction:
+      type: object
+      required:
+        - id
+        - inputs
+        - dataInputs
+        - outputs
+      properties:
+        id:
+          type: object
+          $ref: '#/components/schemas/ModifierId'
+        inputs:
+          type: array
+          items:
+            $ref: '#/components/schemas/ErgoTransactionInput'
+        dataInputs:
+          type: array
+          items:
+            $ref: '#/components/schemas/ErgoTransactionDataInput'
+        outputs:
+          type: array
+          items:
+            $ref: '#/components/schemas/ErgoTransactionOutput'
+
+    Header:
```

Comment:
There is `BlockHeader` which is already defined. If structure of the object described above is different please make It clear (encode it in the naming somehow).

Reply:
"Header" and "BlockHeader" definitions  looks very similar.
ErgoLikeContext refers Header class and it is decoded in the following way in JsoncCodes:

`implicit val headerDecoder: Decoder[Header] = { cursor =>
    for {
      id <- cursor.downField("id").as[Coll[Byte]]
      version <- cursor.downField("version").as[Byte]
      parentId <- cursor.downField("parentId").as[Coll[Byte]]
      adProofsRoot <- cursor.downField("adProofsRoot").as[Coll[Byte]]
      stateRoot <- cursor.downField("stateRoot").as[AvlTreeData]
      transactionsRoot <- cursor.downField("transactionsRoot").as[Coll[Byte]]
      timestamp <- cursor.downField("timestamp").as[Long]
      nBits <- cursor.downField("nBits").as[Long]
      height <- cursor.downField("height").as[Int]
      extensionRoot <- cursor.downField("extensionRoot").as[Coll[Byte]]
      minerPk <- cursor.downField("minerPk").as[Coll[Byte]]
      powOnetimePk <- cursor.downField("powOnetimePk").as[Coll[Byte]]
      powNonce <- cursor.downField("powNonce").as[Coll[Byte]]
      powDistance <- cursor.downField("powDistance").as[special.sigma.BigInt]
      votes <- cursor.downField("votes").as[Coll[Byte]]
    } yield new CHeader(id, version, parentId, adProofsRoot, stateRoot, transactionsRoot, timestamp, nBits,
      height, extensionRoot, SigmaDsl.decodePoint(minerPk), SigmaDsl.decodePoint(powOnetimePk), powNonce, powDistance, votes)`

Some fields (powDistance, powNonce, powOnetimePk, minerPk, extensionRoot) are missed in "BlockHeader" definition. This is why I have to add "Header" section to openapi.yaml.
I have no idea how to "encode it in the naming somehow" if class Header really describes block header. I can change existed definition of BlockHeader, but I afraid that it will break some other code. For example it contains "extensionHash" and "extensionId" fields, while Header class contains "extensionRoot".

I still wonder of openapi.yaml is written manually or it is generated automatically from Scala object definitions.